src/inspector_agent.cc

```diff
@@ -380,6 +388,51 @@ void AgentImpl::WaitForDisconnect() {
   inspector_->runMessageLoopOnPause(0);
 }
 
+String16 ToProtocolString(v8::Local<v8::Value> value) {
+  if (value.IsEmpty() || value->IsNull() || value->IsUndefined() ||
+      !value->IsString()) {
+    return String16();
+  }
+  v8::Local<v8::String> string_value = v8::Local<v8::String>::Cast(value);
+  wstring buffer(0, string_value->Length());
+  string_value->Write(&buffer[0], 0, string_value->Length());
+  return String16(buffer);
+}
+
+void AgentImpl::FatalException(v8::Local<v8::Value> error,
+                               v8::Local<v8::Message> message) {
+  if (!IsStarted())
+    return;
+  auto env = parent_env_;
+  v8::Local<v8::Context> context = env->context();
+
+  int script_id = message->GetScriptOrigin().ScriptID()->Value();
+  std::unique_ptr<blink::V8StackTrace> stack_trace =
```

Comment:
Looks like @ofrobots is right that it picks up a non-system std::unique_ptr because this PR builds for me on OS X 10.8.  If you want to restore the other use, go ahead.